core/finality-grandpa/Cargo.toml

```diff
@@ -17,9 +17,10 @@ log = "0.4"
 parking_lot = "0.4"
 tokio = "0.1.7"
 substrate-finality-grandpa-primitives = { path = "primitives" }
+rand = "0.6"
 
 [dependencies.finality-grandpa]
-version = "0.3.0"
+path = "../../../finality-afg"
```

Comment:
I don't think you wanted to commit that.

Reply:
(i did, because it's inprogress :P)